hw/ip/lc_ctrl/data/lc_ctrl_testplan.hjson

```diff
@@ -12,21 +11,135 @@
+
+            **Checks**:
+            - Check if status register reflects the correct error bit
+            - Check if lc_program_failure alert is triggered
+            - Check if lc_state moves to escalation state
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: lc_state_failure
+      desc: '''
+            This test checks lc_state failure by:
+            - Driving invalid data to lc_ctrl input `otp_lc_data_i` fields `lc_state` and `lc_cnt`
+            - Backdoor changing lc_ctrl FSM's to invalid value
+            For invalid value, the testbench will test using random value and valid `A/B/C/D`
+            values with different orders.
+
+            **Checks**:
+            - Check if status register reflects the correct error bit
+            - Check if lc_state_failure alert is triggered
+            - Check if lc_state moves to escalation state
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: lc_errors
+      desc: '''
+            This test randomly executes the error senarios:
+            - Otp_ctrl input lc_trans_cnt reaches 16
+            - Lc_ctrl state transition request is invalid
+            - Input LC token does not match the output from otp_ctrl
+            - Flash rma responses to lc_ctrl request with error
+            - LC clock bypass responses with error
+            - Input otp_lc_data's error bit is set to 1
+            Note that all the above scenarios except the last one requires a reset to recover.
+
+            **Checks**:
+            - Check if status register reflects the correct error bit
+            - Check if lc_state moves to correct exit state
+            - Check if lc_trans_cnt is incremented
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: security_escalation
+      desc: '''
+            This test checks two security escalation responses:
+            1. Wipe secrets: permanently asserts lc_escalate_en signal
+            2. Scrap state: lc_ctrl moves to escalation state, check the state will be cleared up
+               upon next power cycle
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: jtag_access
+      desc: '''
+            This test checks jtag debug interface in lc_ctrl.
+            This test will use both JTAG TAP and TLUL to access the CSR space.
+            All above CSR sequences should be accessible via both interfaces.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: jtag_priority
+      desc: '''
+            This test covers a corner case in JTAG and TLUL interfaces.
+
+            **Stimulus**:
+            - Issue transition requests from TLUL and JTAG interfaces at the same time
+
+            **Checks**:
+            - Ensure TAP interface has the priority
+            - Once request is completed, ensure the non-prioritized interface's CSR reading are all
```

Comment:
It's not entirely clear what the "request" refers to here, so it may make sense to clarify this part. I think we'd want to ensure that right after the "mutex claim" operation, the non-prioritized interface returns 0 from the CSRs.

Reply:
Good point Michael, let me amend that.